sites/shared/components/workbench/index.mjs

```diff
@@ -23,7 +23,7 @@ const defaultUi = {
 
 const draftViews = ['draft', 'test']
 
-export const Workbench = ({ design, Design, baseSettings, DynamicDocs, from }) => {
+export const Workbench = ({ design, Design, baseSettings, DynamicDocs, from, set }) => {
```

Comment:
Oops. meant to undo that. 

Separately, can you point me to where measurements are being merged with base settings? I understand *why* that's supposed to happen, I just don't see that it is happening, and when I try to make a new pattern from cset in your branch, nothing I get an empty view because there are no measurements